include/cantera/thermo/PengRobinson.h

```diff
@@ -248,7 +254,7 @@ class PengRobinson : public MixtureFugacityTP
      */
     double m_a;
 
-    //! Value of \f$\alpha\f$ in the equation of state
+    //! Value of \f$\a alpha\f$ in the equation of state
```

Comment:
```suggestion
    //! Value of \f$a \alpha\f$ in the equation of state
```

Reply:
Ahh I thought I fixed that typo before committing! I'm not as familiar with the review process, do I go ahead and commit the suggestion and squash it later?

Reply:
You can commit the suggestion and squash it / do a fix up or just update it in one of your own commits (and still squash / fix up)

Reply:
Perfect. I'll wait to squash it until the other reviews are done so that I can do it all in one. Thanks!